lib/expand.js

```diff
@@ -747,6 +747,20 @@ function _expandObject({
     });
   }
 
+  // @value must not be an object or an array (unless framing) or if @type is @json
+  if('@value' in element) {
+    const value = element['@value'];
+    if(element['@type'] === '@json' && _processingMode(activeCtx, 1.1)) {
+      // Sllow any value, to be verified when the object is fully expanded and the @type is @json.
```

Comment:
```suggestion
      // allow any value, to be verified when the object is fully expanded and the @type is @json.
```

Reply:
It would be better to combine these conditionals rather than leave this empty one as it's not really a style we use anywhere else.